jaeger-zipkin/build.gradle

```diff
@@ -1,12 +1,16 @@
 description = 'Integration library for Zipkin'
 
 dependencies {
-    compile group: 'io.zipkin.reporter', name: 'zipkin-reporter', version: '1.1.2'
-    compile group: 'io.zipkin.reporter', name: 'zipkin-sender-urlconnection', version: '1.1.2'
+    compile group: 'io.zipkin.reporter', name: 'zipkin-reporter', version: '1.0.4'
```

Comment:
I had to downgrade this because of some nasty classpath conflicts between the version of Zipkin that 1.1.2 and 2.6.0 of the reporter were using.  Is there some way to pin the version of Zipkin used by dependencies of this module?

Reply:
What If you exclude the old zipkin(2.2.1) from reporter 1.1.2?

Reply:
I'll see if it is still an issue after I revert the ZipkinSender and only implement the V2 Reporter.